app/src/test/java/com/github/barriosnahuel/vossosunboton/feature/playback/PlayerControllerTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.github.barriosnahuel.vossosunboton.feature.playback
+
+import android.media.MediaPlayer
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.verifySequence
+import org.junit.Test
+
+class PlayerControllerTest {
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicFunction - [on stopPlayingSound when media player is playing should stop it] at /src/app/src/test/java/com/github/barriosnahuel/vossosunboton/feature/playback/PlayerControllerTest.kt:12:5](https://app.codacy.com/app/barrios.nahuel/push-me/pullRequest?prid=4481303)